docs/update_tails_usbs.rst

```diff
@@ -0,0 +1,123 @@
+Updating Tails USBs
+===================
+
+There are 3 components of SecureDrop that use Tails USBs:
+
+  #. The *Admin Workstation* (allowed to connect to the Internet)
+  #. The *Journalist Workstation* (allowed to connect to the Internet)
+  #. The *Secure Viewing Station* (airgapped, never allowed to connect to any network)
+
+You are responsible for keeping your Tails USBs updated. This guide will
+explain that process.
+
+One-Click Updates
```

Comment:
Agree - I wasn't really sure how to indicate that you just start the install by clicking "yes" to the dialog box (as opposed to using cli or anything), but I don't like the Tails' phrasing of 'automatic' because it isn't automatic and people may associate that with not having to take any action.